db/models/Comment.js

```diff
@@ -0,0 +1,21 @@
+
+
+module.exports = (sequelize, DataTypes) => {
+  const Comment = sequelize.define('Comment', {
+    userId: DataTypes.INTEGER,
+    articleId: DataTypes.INTEGER,
+    content: DataTypes.STRING
+  }, {});
+  Comment.associate = (models) => {
+    Comment.belongsTo(models.User, {
+      foreignKey: 'userId',
+      as: 'user'
```

Comment:
can see you did not cascade the association of comment to users the way you did in the case of Articles on line 17, please can you explain why?

Reply:
Thank you for your feedback, I actually did, kindly expand the file for more detail.